crates/subspace-node/src/commands/run/consensus.rs

```diff
@@ -517,9 +526,9 @@ pub(super) fn create_consensus_chain_configuration(
         mut sync,
     } = consensus_node_options;
 
+    init_logger(json_logging);
```

Comment:
Unfortunately we need to init the logger before we process the config, so we can log any errors in the config. Have you considered having two different tracing subscribers instead? (One for stderr, and the other for json?)

Reply:
I didn't saw any logs, but probably a mistake from my side. I will rework this part as well. 

I wanted to init the logger anywaysand then switch to the desired mode, but that was not fine. Will will rework